openedx/features/lums/admin.py

```diff
@@ -0,0 +1,16 @@
+from django.contrib import admin
+
+from openedx.features.lums.models import UserProfileExtension
+
+
+class UserProfileExtensionAdmin(admin.ModelAdmin):
+    """ Admin interface for the UserAttribute model. """
```

Comment:
I think there is a typo, this is for `UserProfileExtension`

Reply:
Done